frontend/demo/component/avatar/avatar-menu-bar.ts

```diff
@@ -1,12 +1,13 @@
 import '../../init'; // hidden-full-source-line
+import '@vaadin/flow-frontend/menubarConnector'; // hidden-full-source-line
```

Comment:
Adding `contextMenuConnector` seems fix the issue with the example:

```suggestion
import '@vaadin/flow-frontend/menubarConnector'; // hidden-full-source-line
import '@vaadin/flow-frontend/contextMenuConnector'; // hidden-full-source-line
```

Reply:
Fixed!